requirements-3.8.txt

```diff
@@ -0,0 +1,10 @@
+numpy==1.21.6
+matplotlib==3.2.2
+pytest==3.6.4
+pylint>=2.15.10
```

Comment:
I _think_ it makes sense to put the less constrained (>=) requirements after the more constrained (==) requirements.